.cirrus.yml

```diff
@@ -0,0 +1,14 @@
+task:
+  name: Build sbtn for macOS ARM64
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-ventura-base:latest
+  coursier_cache:
+    folder: $HOME/Library/Caches/Coursier/v1
+  sbt_cache:
+    folder: $HOME/.sbt
+  build_script: arch -arm64 ./.cirrus/publish-macos-silicon.sh
+  binary_artifacts:
+    name: Binary assets
+    path: "sbt/client/target/bin/sbtn"
```

Comment:
We always build the binary and export it as a downloadable asset from the Cirrus CI UI.

We also set up a few commands in the script `publish-macos-silicon.sh` that should publish the binary as a release asset, when the job is triggered by a release (I could not test that, though).